plenum/common/config_util.py

```diff
@@ -48,8 +48,7 @@ def extend_with_default_external_config(extendee: object,
 
     # fail if network is not set
     if not extendee.NETWORK_NAME:
-        return
-        # raise Exception('NETWORK_NAME must be set')
+        raise Exception('NETWORK_NAME must be set')
```

Comment:
Can we provide more detailed information about where `NETWORK_NAME` must be set?